packages/hooks/src/useControlledValue/useControlledValue.spec.ts

```diff
@@ -16,7 +16,7 @@ describe('packages/lib/useControlledValue', () => {
 
     act(() => {
       current.handleChange({ target: { value: 'banana' } } as ChangeEvent<any>);
-      current.setInternalValue('banana');
+      current.setUncontrolledValue('banana');
```

Comment:
We should add more tests to this. 
The current one should be named e.g. "`handleChange` calls provided callback".

We should create another test where we update the value passed into the hook and expect that the current.value changes